src/test/scala/mesosphere/marathon/core/deployment/impl/DeploymentPlanRevertTest.scala

```diff
@@ -338,11 +338,11 @@ class DeploymentPlanRevertTest extends UnitTest with GroupCreation {
             createGroup("othergroup2".toAbsolutePath),
             createGroup("othergroup3".toAbsolutePath),
             {
-              val id = "withdeps".toAbsolutePath // withdeps still exists because of the subgroup
+              val id = "/withdeps".toAbsolutePath // withdeps still exists because of the subgroup
               createGroup(
                 id,
                 apps = Group.defaultApps,
-                groups = Set(createGroup(id / "some")),
+                groups = Set(createGroup((id / "some").asAbsolutePath)),
```

Comment:
Same as with AbsolutePath.root, AbsolutePath./ should directly return an absolute path again, without calling asAbsolutePath.

Reply:
Man, I thought I could not override `./` and `append` because of the return type. I was wrong. This makes thinks much simpler. Thanks.